### PR TITLE
DOCS-2901: Edit fleet management code samples

### PR DESCRIPTION
**Pull request by sguequierre**
* edits made from qa

### CODE REVIEW

File: static/include/app/apis/generated/app.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Return details about the requested organization.
 
 **Parameters:**
 
-- `org_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the organization to query.
+- `org_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the organization to query. You can retrieve this from the organization settings page.
 
 **Returns:**
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:143]: Can you change `The requested organization.` to something like
`The organization object for the requested organization containing the organizaiton's id, name, public namespace, and more.`.

I think that might be helpful

File: static/include/app/apis/generated/app.md [hunk 2]
```diff
@@ -323,19 +339,21 @@ If an invitation has only one authorization and you want to remove it, delete th
...
-    resource_type="abc",
-    resource_id="resource-identifier123",
-    identity_id="id12345",
-    organization_id="org_id_123"
+auth = Authorization(
+    authorization_type="role",
+    authorization_id="location_owner",
+    resource_type="location", # "robot", "location", or "organization"
+    # machine id, location id or org id
+    resource_id="012456lni0",
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:347]: Consistency with line above
```suggestion
     resource_id="012456lni0", # machine ID, location ID, or org ID
```

File: static/include/app/apis/generated/app.md [hunk 3]
```diff
@@ -899,7 +921,8 @@ Get the logs associated with a specific machine {{< glossary_tooltip term_id="pa
 
 ```python {class="line-numbers linkable-line-numbers"}
 part_logs = await cloud.get_robot_part_logs(
-    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22", num_log_entries=20)
+    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22", num_log_entries=20
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:924]: if the closing parentheses is on a new line, generally you style it like thi
```suggestion
    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22",
    num_log_entries=20
```

File: static/include/app/apis/generated/app.md [hunk 4]
```diff
@@ -1614,15 +1649,16 @@ Changes an existing role to a new role.
...
 ```python {class="line-numbers linkable-line-numbers"}
 await cloud.change_role(
-    organization_id="organization-id",
+    organization_id="<YOUR-ORG-ID>",
     old_identity_id="abc01234-0123-4567-ab12-a11a00a2aa22",
     old_role="operator",
     old_resource_type="location",
     old_resource_id="111ab12345",
     new_identity_id="abc01234-0123-4567-ab12-a11a00a2aa22",
-    new_role="owner",s
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:1623]: nice catches

File: static/include/app/apis/generated/app.md [hunk 5]
```diff
@@ -1770,7 +1807,7 @@ Get registry item by ID.
 
 **Parameters:**
 
-- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item.
+- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. This is the namespace and name of the item in the form `namespace:name`. For example, [Viam's `csi-cam-pi` module's](https://app.viam.com/module/viam/csi-cam-pi) item ID would be `viam:csi-cam-pi`.
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:1810]: This is the case for public items, not for private ones. 
```suggestion
- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. For public items, this is the namespace and name of the item in the form `namespace:name`. For example, [Viam's `csi-cam-pi` module's](https://app.viam.com/module/viam/csi-cam-pi) item ID would be `viam:csi-cam-pi`. For private items, it is `org-id:name`. For example, `abc01234-0123-4567-ab12-a11a00a2aa22:training-script`.
```

File: static/include/app/apis/generated/app.md [hunk 6]
```diff
@@ -1836,6 +1875,9 @@ Update a registry item.
 **Example:**
 
 ```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.packages import PackageType
+from viam.proto.app import Visibility
+
 await cloud.update_registry_item("item-id", PackageType.PACKAGE_TYPE_ML_TRAINING, "description", Visibility.VISIBILITY_PUBLIC)
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:1881]:
```suggestion
await cloud.update_registry_item(
    "your-namespace:your-name",
    PackageType.PACKAGE_TYPE_ML_TRAINING,
    "description",
    Visibility.VISIBILITY_PUBLIC
)
```

File: static/include/app/apis/generated/app.md [hunk 7]
```diff
@@ -1879,7 +1946,7 @@ Delete a registry item.
 
 **Parameters:**
 
-- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item.
+- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. This is the namespace and name of the item in the form `namespace:name`.
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:1949]: can you change the `"item-id"` in the code sample to `"your-namespace:your-name"`, just to help

File: static/include/app/apis/generated/app.md [hunk 8]
```diff
@@ -1981,12 +2057,23 @@ Upload a {{< glossary_tooltip term_id="module" text="module" >}} file.
 
 **Returns:**
 
-- ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)): ID of uploaded file.
+- ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)): URL of uploaded file.
```
Comment (npentrel) [inline static/include/app/apis/generated/app.md:2060]: good catch

**Review by npentrel — COMMENTED** [review]
This looks good. Some small comments. Only major one is question around `GetModule `

File: static/include/app/apis/generated/app.md (hunk 5 above)
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:1810]: oh didn't catch that, thank you!

File: static/include/app/apis/generated/app.md (hunk 1 above)
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:143]: done

File: static/include/app/apis/generated/app.md (hunk 3 above)
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:924]: done

File: static/include/app/apis/generated/app.md [hunk 9]
```diff
@@ -2003,7 +2090,7 @@ Get a {{< glossary_tooltip term_id="module" text="module" >}} by its ID.
 
 **Parameters:**
 
-- `module_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): ID of the module being retrieved, containing module name or namespace and module name.
+- `module_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): ID of the module being retrieved, containing namespace and module name.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:2093]: I get grpclib.exceptions.GRPCError: (<Status.INVALID_ARGUMENT: 3>, 'ModuleID "asdf" is invalid. It must be "prefix:modulename" where prefix is either an orgid or a namespace', None)

[the remark this answers is not in the document]
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:2093]: I just tested and either prefix can work for private modules (org id and namespace), I don't have any public modules but i'm assuming it works with both prefixes for both ways-- updating to reflect

[the remark this answers is not in the document]
Comment (sguequierre) [reply static/include/app/apis/generated/app.md:2093]: ok tested with public models in viam-dev org and can confirm both prefixes work either way

**Review by npentrel — APPROVED** [review]

**Pull request merged**